packages/material-ui/src/DialogActions/DialogActions.js

```diff
@@ -11,6 +11,7 @@ export const styles = {
     alignItems: 'center',
     padding: 8,
     justifyContent: 'flex-end',
+    flex: '0 0 auto',
```

Comment:
The default flex value is `0 1 auto`. We are using `flex-shrink: 0;` at many different occasions in the past. I believe we only need to change this value. Why using flex instead?

```suggestion
    flexShrink: 0,
```

Reply:
The original problem was caused (I assume) by an incorrect assumption about the defaults, so the change is explicit.
Also, for consistency with the way DialogTitle and DialogContent are styled.